packages/tpetra/core/cmake/Dependencies.cmake

```diff
@@ -1,5 +1,6 @@
 TRIBITS_PACKAGE_DEFINE_DEPENDENCIES(
   LIB_REQUIRED_PACKAGES Teuchos TpetraClassic KokkosCore KokkosContainers KokkosAlgorithms TeuchosKokkosCompat TeuchosKokkosComm KokkosKernels
-  LIB_OPTIONAL_PACKAGES Epetra TpetraTSQR
+  LIB_OPTIONAL_PACKAGES Epetra EpetraExt TpetraTSQR
```

Comment:
OK for now, but let's think about decoupling Tpetra from EpetraExt in the future.